pallets/pallet-asset-switch/README.md

```diff
@@ -1 +1,129 @@
-WIP.
+# Asset switch pallet
+
+The asset switch pallet introduces the possibility for the chain local currency to be switched 1:1 with a remote asset at a remote destination, according to the provided configuration, using XCM.
+
+This is possible via the creation of a *switch pair*, which contains information about the identifier of the remote asset (e.g., its `MultiLocation`), the remote location on which the asset lives (and with which XCM communication takes place), the circulating supply of the remote asset which can be switched back for the local currency, and additional information relevant for the XCM communication, which are explained more in depth later on.
+
+## In a gist
+
+The pallet lets users on the source chain lock `N` tokens of the chain's native currency into a switch pair specific account, to receive `N` remote assets on the configured remote location, which are transferred to them from the source chain's sovereign account at destination.
+Then, the chain, which i
+
+## Design choices
```

Comment:
Not sure, if it is important, but we can also mention maybe that the pallet is using XCM version 3

Reply:
Mhh I think that's implementation details we can leave out, as it will change when we upgrade our dependencies.